bot/cogs/eval.py

```diff
@@ -181,10 +184,12 @@ def _format(self, inp, out):  # (str, Any) -> (str, discord.Embed)
         if code.startswith("py\n"):
             code = "\n".join(code.split("\n")[1:])
 
-        if not re.search(  # Check if it's an expression
-                r"^(return|import|for|while|def|class|"
-                r"from|exit|[a-zA-Z0-9]+\s*=)", code, re.M) and len(
-                    code.split("\n")) == 1:
+        if (
+            not re.search(  # Check if it's an expression
+                r"^(return|import|for|while|def|class|" r"from|exit|[a-zA-Z0-9]+\s*=)", code, re.M
+            )
+            and len(code.split("\n")) == 1
+        ):
```

Comment:
this if is pretty hideous both before and after the change. perhaps we can refactor it to something altogether better.